desktop/tests/test_chat.py

```
@@ -0,0 +1,377 @@
+    assert res.status_code == 200
+    body = res.json()
+    assert body["slice"] == 29
+    assert body["model"] == "fake"
+    assert body["piece"] == "sidecar"
+
+
+def test_ws_rejects_missing_token(tmp_path):
+    client = TestClient(app(tmp_path))
+    try:
+        with client.websocket_connect("/ws/chat"):
+            assert False, "expected reject"
```

Ruff (0.16.2)

[warning] Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)